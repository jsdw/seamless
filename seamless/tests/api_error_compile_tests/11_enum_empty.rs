#[derive(seamless::ApiError)]
#[api_error(internal)]
enum Foo { }

fn main () {}