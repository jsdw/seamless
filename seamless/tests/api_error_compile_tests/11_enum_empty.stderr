error: ApiError: Enums without variants are not supported
 --> $DIR/11_enum_empty.rs:3:6
  |
3 | enum Foo { }
  |      ^^^
